SU-HSDO/suhumsci
# Image styles fix

# READY FOR REVIEW

Please test thoroughly. @kaseybon, can you do a full review, and then once you approve, let's have @pookmish look before it gets merged.

## Summary
> Previously our Large and Medium image styles used to scale (rather than crop square). In the new image styles, they are cropping square. When we switch to the new image styles, existing sites will look significantly different.

> Can we switch the new stanford_image_large and stanford_image_medium to the scaled versions and rename the square versions stanford_image_[size]_square?

(see https://sparkbox.atlassian.net/browse/STN-264 for more details)

## Need Review By (Date)
asap

## Urgency
'Medium'

## Steps to Test
1. Pull down this branch
2. Import all the configuration for these image styles and view modes.
    - `lando drush @mysite.local config-import`
    - Warning: this could impact your local environment, if you have different local config.
3. Go to a basic page, create a Text Area Paragraph, insert each of the image styles and confirm that they work as expected (including with floats, etc)
4. Check for regressions.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
